### ISSUE TITLE
Economy got wiped?

### ISSUE THREAD

**Issue by NoFailStudios** [MilkBowl/Vault#922]
For some reason my whole server's economy got wiped and no one has money anymore.
Is there a specifc reason of this issue and is there any way to prevent it?

**Comment by Geolykt** [comment]
Not a Vault issue. This is related to the backend. Either you now use a different backend or the economy plugin broke

**Comment by NoFailStudios** [comment]
We don’t use any other economy plugins.

**Comment by Geolykt** [comment]
... then you haven't read the description of what Vault does - no wonder you are getting these issues.

This plugin is an abstract LIBRARY. Libraries don't do anything on their own unless the developers of said library explicitly claim it can do more (they do not in this case). However for reasons beyond me 50% of issues created on this issue page can be traced to  people not (properly) installing an economy provider alongside Vault. The other 50% are people asking about version X support.

**Comment by NoFailStudios** [comment]
Okay I use essentialsX but what plugin stores the data for each player,
EssentialsX or Vault? Like which of them also acts like a DB?

**Comment by Geolykt** [comment]
Vault only provides the abstraction layer so plugins can communicate with each other.

Vault does NOT store data.
Vault does NOT act as a DB.
Vault does NOT register economies.
Vault does NOT provide a fallback economy.
Vault does NOT enforce any DB schemes.

**Comment by NoFailStudios** [comment]
Yeah but that is the issue, it for some reason wiped the server’s economy
when I reloaded a completely different plugin, which has nothing to do with
economy buy does communicate with vault in order to get LuckPerms' data
about playername, prefixes and suffixes. So my theory is because Vault is
like a middleman in between all the plugins, it bugged out.
